frontend/src/routes/Infrastructure/Clusters/ManagedClusters/ManagedClusters.tsx

```diff
@@ -73,7 +73,9 @@ export default function ManagedClusters() {
 
     const onBoardingModalID = `${window.location.href}/clusteronboardingmodal`
     const [openOnboardingModal, setOpenOnboardingModal] = useState<boolean>(
-        localStorage.getItem(onBoardingModalID) ? localStorage.getItem(onBoardingModalID) === 'show' : true
+        localStorage.getItem(onBoardingModalID)
+            ? localStorage.getItem(onBoardingModalID) === 'show'
+            : clusters.length === 1
```

Comment:
@o-farag Can you also check if the one cluster on the list is `local-cluster`? If RBAC is enabled, the user might not have access to see `local-cluster` so if there's a cluster, it might be one that they already created.